frontend/public/components/app.jsx

```diff
@@ -239,6 +239,16 @@ class App extends React.PureComponent {
   }
 }
 
+// Load cached API resources from localStorage. We'll still perform API
+// discovery on load to refresh the cache, but this can greatly speed up
+// initial page load if the resources we need are cached.
+getCachedResources().then(resources => {
```

Comment:
This looks like it is causing a double-loading spinner on the "Search" view: https://jenkins-tectonic.prod.coreos.systems/job/console-build/9435/artifact/frontend/gui_test_screenshots/test-gui-report.html

Reply:
Yeah, thanks. I'll investigate.